src/ai/backend/agent/docker/kernel.py

```diff
@@ -127,6 +132,52 @@ async def get_service_apps(self):
         result = await self.runner.feed_service_apps()
         return result
 
+    async def check_commit_tag(self, commit_path: Path, get_lock: bool = False):
+        container_id: str = str(self.data["container_id"])
+        tag_path = commit_path / "tags" / container_id
+
+        tag_path.parent.mkdir(parents=True, exist_ok=True)
+        try:
+            with open(tag_path, mode="x") as f:
+                if get_lock:
+                    f.write("")
+        except FileExistsError:
+            return int(False)
+        return int(True)
```

Comment:
Why do such non-trivial type conversion?
Make the method name more standing out with the intention and/or use enum or just plain bool values.